<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_NetPortSettings</class>
 <widget class="QDialog" name="QG_NetPortSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网口设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>网口参数</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="lay_param">
          <widget class="QWidget" name="lay_paramPage1" native="true">
           <layout class="QGridLayout" name="_2" columnstretch="0,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>端口</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_cport"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>远程地址</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="le_cip">
              <property name="text">
               <string>127.0.0.1</string>
              </property>
              <property name="placeholderText">
               <string>服务器</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>端口</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_sport"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>本机地址</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cb_sip"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lw_client"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_close">
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_open">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_debug">
       <property name="title">
        <string>调试窗口</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>接收消息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_rep">
            <property name="text">
             <string>自动回复</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_repmsg">
            <property name="text">
             <string>Success</string>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_clr">
            <property name="text">
             <string>清空接收</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="tb_rec">
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>发送消息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_ascii">
            <property name="text">
             <string>Ascii</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="le_cmd">
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>模式定义</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="lay_mode" native="true">
        <layout class="QHBoxLayout" name="_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="rb_client">
           <property name="text">
            <string>客户端</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_server">
           <property name="text">
            <string>服务器</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="lay_server" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>通讯协议选择</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cb_proto">
           <item>
            <property name="text">
             <string>字符流</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Modbus</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自定义</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>指令字符</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_encode">
             <item>
              <property name="text">
               <string>ASCII</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Unicode</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>标刻完成输出</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_sign">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>默认</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自定义</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>标刻内容</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="le_sigh"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>执行完成输出</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cb_finish">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>默认</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自定义</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="le_finish"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>错误警示输出</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cb_err">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>无</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>默认</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>自定义</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="le_err"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dbb_over">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbb_over</sender>
   <signal>accepted()</signal>
   <receiver>QG_NetPortSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbb_over</sender>
   <signal>rejected()</signal>
   <receiver>QG_NetPortSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
